apps/ui/lib/lens/misc/Inbox/Inbox.tsx

```diff
@@ -122,13 +122,34 @@ const Inbox = ({ channel }: Props) => {
 		<Column pt={2}>
 			<Tabs>
 				<Tab title="Open">
-					<InboxTab query={openQuery} channel={channel} canArchive={true} />
+					<InboxTab
+						query={openQuery}
+						channel={channel}
+						canArchive={true}
+						isDirectMention={false}
+					/>
+				</Tab>
+				<Tab title="Direct Mentions">
+					<InboxTab
+						query={openQuery}
```

Comment:
This query will return a lot of things that are not direct mentions, when I believe you are then filtering on. Are you sure there's not a better query we can use to reduce work on both the backend and frontend (not to mention network payload size).

Reply:
I have an idea on a query that might do the trick. I'll try it either this weekend or Monday and let you know how it goes. 👍

Reply:
Trying my idea here: https://github.com/product-os/jellyfish/pull/9357